modules/collect/src/main/java/com/opengamma/strata/collect/Messages.java

```diff
@@ -110,4 +119,79 @@ public static String format(String messageTemplate, Object... args) {
     return builder.toString();
   }
 
+  /**
+   * Formats a templated message inserting named arguments.
+   * <p>
+   * This method combines a template message with a list of specific arguments.
+   * It can be useful to delay string concatenation, which is sometimes a performance issue.
+   * The approach is similar to SLF4J MessageFormat, Guava Preconditions and String format().
+   * <p>
+   * The message template contains zero to many "{name}" placeholders.
+   * Each placeholder is replaced by the next available argument.
+   * Empty "{}" placeholders do not get replaced, and will be present in the output in the same form.
```

Comment:
No. `{}` is still a placeholder, just an unamed one.

Reply:
That argument is outdated. Both named and unnamed placeholders are not replaced, as expected